### ISSUE TITLE
[BUG] - No  separators in a *.gdscript file starting with a comment!

### ISSUE THREAD

**Issue by BrianOSH** [alefragnani/vscode-separators#104]
```
No  separators in a *.gdscript file starting with a comment! unless a class_name is added to the file.

##----------------------------
## No separators
##----------------------------
extends Button

##----------------------------
## Separators
##----------------------------
class_name Name extends Button


- Extension version: 2,7,0
- VSCode version: v1.95.3
- OS version: Linux Mint 5.15.0-126-generic

```

**Comment by alefragnani** [comment]
You either use the `Explorer: Focus on Outline View` command, via Command Palette, or look for it in the bottom of the (native) Explorer Panel, near the Timeline view.

**Comment by alefragnani** [comment]
BTW, which language extension are you using?

**Comment by alefragnani** [comment]
Issue opened in language extension repo: https://github.com/godotengine/godot-vscode-plugin/issues/759
Considered duplicate of https://github.com/godotengine/godot-vscode-plugin/issues/658
Considered issue in the LSP https://github.com/godotengine/godot/issues/98164
MR that fixes the issue https://github.com/godotengine/godot/issues/98164

I suppose once the MR is merged, and a new release is published, the separators should work just fine.
